wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WTabSet.java

```diff
@@ -281,11 +314,22 @@ public void addSeparator() {
 	 * Adds a tab to the tab set.
 	 *
 	 * @param tab the tab to add.
-	 * @deprecated use {@link #addTab(WComponent, String, TabMode)}
+	 * @deprecated use {@link #addTab(WComponent, String)} or overloaded version.
 	 */
 	@Deprecated
 	public void add(final WTab tab) {
+		addTab(tab);
+	}
+
+	/**
+	 * Adds a {@link WTab} to the tab set.
+	 *
+	 * @param tab the tab to add.
+	 * @return the tab which was added to the tab set.
+	 */
+	private WTab addTab(final WTab tab) {
```

Comment:
As the Constructor for `WTab` is `protected`, the `addTab` method should probably be `protected` as well, which would allow the class to be extended. 
What do you think?